main/antivirus/stubs/freshclam.profile.mas

```diff
@@ -2,6 +2,7 @@
   $observerScript
 </%args>
 /run/clamav/clamd.ctl rw,
+/proc/*/status r,
```

Comment:
Added https://github.com/Zentyal/zentyal/commit/378df195971f62c193c47c9c91093ab13639f081 sorry for the rush but this was breaking several suites in the tests.